signup/index.php

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+<html class="no-js" lang="">
+  <head>
+  <title>Signup Test</title>
+  <link href="https://fonts.googleapis.com/css?family=Montserrat" rel="stylesheet">
```

Comment:
I'll probably end up moving these font imports into a main css file... It's faster and helps separate skin from structure.